prow/create-build-cluster.sh

```diff
@@ -54,6 +54,19 @@ ADMIN_IAM_MEMBER="${ADMIN_IAM_MEMBER:-group:mdb.cloud-kubernetes-engprod-oncall@
 # Overriding output
 OUT_FILE="${OUT_FILE:-build-cluster-kubeconfig.yaml}"
 
+
+# Require bash version >= 4.4
+if [[ -z "${BASH_VERSINFO[@]}" ]]; then
```

Comment:
this should never happen in bash ™️

Reply:
I think I saw somewhere that it was added in bash 2.0, but that was ~25 years ago :upside_down_face: